snap/info.go

```diff
@@ -124,6 +124,7 @@ type SideInfo struct {
 	EditedSummary     string   `yaml:"summary,omitempty" json:"summary,omitempty"`
 	EditedDescription string   `yaml:"description,omitempty" json:"description,omitempty"`
 	Private           bool     `yaml:"private,omitempty" json:"private,omitempty"`
+	SupportURL        string   `yaml:"support-url,omitempty" json:"support-url,omitempty"`
```

Comment:
I have mixed feelings about this approach (get it from store and save in state), for one it means snaps on images will not get this unless we extend prepare-image and seed